js/src/helpers/taxonomy-replacevars.js

```diff
@@ -202,6 +202,11 @@ class TaxonomyReplacementVariables {
 	 */
 	listen() {
 		getPostType().then( postType => {
+			// Pages don't have taxonomies.
```

Comment:
That's no right. Page can have taxonomies but you have to register them.

See:
https://codex.wordpress.org/Function_Reference/register_taxonomy_for_object_type